### ISSUE TITLE
Adding torrent via Torrent Control makes Transmission Web Control unresponsive.

### ISSUE THREAD

**Issue by KosVas91** [Mika-/torrent-control#148]
The title is self-explanatory, but I will add some details.
I am using Torrent Control with FF 112.0, to add torrents to Transmission 4.0.2 instance running on OpenWRT (SNAPSHOT (r22571-00a240e77f)) router. Also, I using Transmission Web Control (2021-09-25-4b2e1858-3) as WebUI.

Issue description:
After adding any torrent via Torrent Control, Transmission Web Control becomes unresponsive, it looks like it lost connection to the transmission daemon (empty torrent list, no speed/free space info). But the torrent itself is added correctly, it can be confirmed with any other Remote GUI or Default Transmission WebUI.
The only way to make WebUI responsive again is to close/reopen FF. After that you can open WebUI and it is working fine until next torrent added via Torrent Control.

I have tested few possible combinations:
Torrent Control + Transmision Default WebUI - OK
Torrent Control + Transmission Web Control - Fail
Transmitter for Transmission + Transmision Default WebUI - OK
Transmitter for Transmission + Transmission Web Control - OK

**Comment by Mika-** [comment]
With Transmission Web Control you propably mean [ronggang/transmission-web-control](https://github.com/ronggang/transmission-web-control)?
Torrent control doesn't have support for that. Web control hasn't been updated in over 5 years, so it's pretty much abandon ware. I won't add support for it, as it may break any time Transmission changes it's api.
Your problem also sounds a little concerning, as it indicates security problem in Web Control if invalid api payload is received.

**Comment by KosVas91** [comment]
Thank you for fast reply! 
Yes, that is Transmission Web Control I spoke about. I am completely new to OpenWRT and Transmission and I chose this WebUI only because it had prebuilt package in OpenWRT repo. Package itself was dated 2021, so I did not pay attention that it is not maintained anymore.
I completely understand you position about not adding support of it to Torrent Control, so I think this issue can be closed.
